packages/evolution-common/src/utils/helpers.ts

```diff
@@ -697,3 +699,140 @@ export const calculateSurveyCompletionPercentage = ({
     );
     return completionPercentage;
 };
+
+/**
+ * Retrieve a visited place and its associated person in the household by the visited place ID.
+ *
+ * @param {string} visitedPlaceId - The ID of the visited place to find.
+ * @param {string|null} [attributePrefix=null] - Optional prefix for visited places attributes.
+ * @param {UserInterviewAttributes} interview - The interview object containing household data.
+ * @returns {{ person: object, visitedPlace: object } | null} - The person and visited place object, or null if not found.
+ */
+export const getVisitedPlaceAndPersonInHouseholdById = function (visitedPlaceId, attributePrefix = null, interview) {
```

Comment:
Remove those for now, some of them are already defined in the `odSurvey/helpers.ts` file. Others can be added as necessary